internal/pkg/generator/forwarder/generator.go

```diff
@@ -3,7 +3,7 @@ package forwarder
 import (
 	"fmt"
 
-	yaml "gopkg.in/yaml.v2"
+	yaml "sigs.k8s.io/yaml"
```

Comment:
any test case to check the effect of this change?

Reply:
This will test it https://github.com/openshift/cluster-logging-operator/pull/946/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R161